src/string/mod.rs

```diff
@@ -320,7 +320,7 @@ unsafe impl StorageMut for BytesMut {
 /// For technical reasons, both are implemented in one go as this type. For the same reason, most
 /// of the documentation can be found here. Users are expected to use the [Str] and [StrMut]
 /// instead.
-#[derive(Copy, Clone, Default)]
+#[derive(Clone, Default)]
```

Comment:
> is there a reason not to have it available for eg. StrInner<&str>?

It is certainly useful for `StrInner<&str>`, but  honestly I don't see the point of it.

> Apart from that being a breaking change (which is OK-ish with a crate that is still 0.something, but a bit annoying anyway),

IMHO `Storage` and `StorageMut` should probably be sealed and only implemented on `Bytes` and `BytesMut`.
I see it as a way to avoid duplicate code.

Reply:
Yeh, I don't see the point of _not_ having it, I tend to implement all the impls that kind of make sense if possible. But whatever :shrug: